test/test_normal_exceptions.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+
+
+def test_simple(tracer, check_golden_file):
+    tracer.start()
+
+    class CustomException(Exception):
+        pass
+
+    a = ImportError("A")
```

Comment:
I think `ImportError("A")` is equivalent to `ImportError()` in our case? In other words, `"A"` and `"B"` are merely distractions, I'd probably remove them from all tests.

Also, having `print(a)` doesn't seem very useful either.